A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js

```diff
@@ -216,28 +248,27 @@ var A320_Neo_LowerECAM_APU;
 
         getAPUN() {
             return SimVar.GetSimVarValue("APU PCT RPM", "percent");
-            
         }
 
-        //Calculates the APU EGT Based on the RPM
+        //Calculates the APU EGT Based on the RPM, APU now reaches peak EGT of 765'C
         getAPUEGTRaw(startup) {
             var n = this.getAPUN();
             if (startup) {
                 if (n < 10) {
                     return 10;
                 } else if (n < 16) {
-                    return (135*n)-1320;
-                } else if (n < 20) {
-                    return -1262 + (224*n) - (5.8 * (n*n));
-                } else if (n < 36) {
-                    return ((-5/4)*n) + 925;
+                    return (135*n)-1600;
+                } else if(n < 20){
+                    return (20*n)+105;
+                } else if (n < 37) {
+                    return (17.125*n)+148.5;
```

Comment:
Seems like some white space is missing here, might wanna check formatting settings in your editor :)